style/checkstyle.xml

```diff
@@ -275,7 +275,9 @@
         <module name="EmptyStatement" />
 
         <!-- Checks for Java Docs. -->
-        <module name="JavadocStyle"/>
+        <module name="JavadocStyle">
```

Comment:
I prefer these in separate PRs

Reply:
You mean separate PR just for Checkstyle? 
This check's false-positives were very annoying, thus made me switch it off.

Reply:
yes. checkstyle changes is something that needs separate attention. it can be a `MINOR` PR. but separate nonetheless